app/access/admin.py

```diff
@@ -1,8 +1,20 @@
 from django.contrib import admin
+from django.db import models
+from django.http import HttpRequest
 
 from .models import User
 
 
 @admin.register(User)
 class UserAdmin(admin.ModelAdmin):  # type:ignore[type-arg]
     '''Admin View for User'''
+    list_display = ('provider', 'username', 'deleted_at')
+    actions = ["make_disabled"]
+
+    @admin.action(description="Disable selected users")
+    def make_disabled(self, request: HttpRequest, queryset: models.QuerySet[User]) -> None:
```

Comment:
Why "make_disabled" instead of "disable"?

Reply:
Not sure what I was thinking. Probably not a lot... will change it.